src/pytorch_lightning/utilities/data.py

```diff
@@ -49,6 +49,18 @@
 warning_cache = WarningCache()
 
 
+class _WrapAttrTag(LightningEnum):
+    SET = "set"
+    DEL = "del"
+
+    def __call__(self, *args):
+        if self == self.SET:
+            fn = setattr
+        else:
```

Comment:
elif?

Reply:
Not needed since it's an `Enum` with only two possible values. It cannot be constructed with any other value